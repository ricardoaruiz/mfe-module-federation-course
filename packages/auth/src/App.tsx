import React from 'react'

const App: React.FC = () => {
  return <h1>Auth works fine!!</h1>
}

export default App
