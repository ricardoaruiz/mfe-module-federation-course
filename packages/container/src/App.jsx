import React from 'react'

const App = () => {
  return (
    <h1>Container works fine!! Now with React</h1>
  )
}

export default App