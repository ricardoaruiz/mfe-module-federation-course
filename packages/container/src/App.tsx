import React from 'react'

const App: React.FC = () => {
  return <h1>Container works fine!! Now with React and Typescript</h1>
}

export default App
