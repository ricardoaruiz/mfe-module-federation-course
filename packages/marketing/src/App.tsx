import React from 'react'

const App: React.FC = () => {
  return <h1>Marketing works fine!1</h1>
}

export default App
