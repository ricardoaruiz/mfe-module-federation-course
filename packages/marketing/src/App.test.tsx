/* eslint-disable testing-library/no-node-access */
import React from 'react'
import { render } from '@testing-library/react'
import { createMemoryHistory } from 'history'

import App from './App'

describe('App', () => {
  it('should render correctly', () => {
    const { container } = render(<App history={createMemoryHistory()} />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
