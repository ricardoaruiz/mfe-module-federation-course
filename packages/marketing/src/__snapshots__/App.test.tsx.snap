// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<h1>
  Marketing works fine!1
</h1>
`;
